### PR TITLE
Binary Search

### LINKED ISSUE #1431: Binary Search
# Aim :- Binary Search

- Write Binary Search Algorithm in Python
- 

### Programming language

- [ ] C
- [ ] C++
- [ ] Java
- [x] Python

Please Assign me this Task under LGM-Soc-21

### PR DESCRIPTION
**Pull request by Ashu-1309**
# Binary Search
The code will be used for binary search .

Fixes #1431 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] Documentation Update

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have created a helpful and easy to understand `README.md`
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

File: Algorithms/Binary_search.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+   2nd mid = 7 , then value[7] = 20
+                    value = 20
+                    Number is found
+
+'''
+
+
+
+
+
```
Comment (sameeksharl) [inline Algorithms/Binary_search.py:41]: remove unused lines

**Comment by sameeksharl** [comment]
@Ashu-1309 if no is not present in the list , then it doesn't print "not found". Do change that
and also take input from user.

**Comment by sameeksharl** [comment]
@Ashu-1309 look into this as well,
 if number is not present in the list , then it doesn't print "not found". Do change that.

**Comment by Amit366** [comment]
Since no update , I am closing it

**Pull request closed without merging**